spec/puppetserver/ca/action/prune_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
+require 'utils/ssl'
+require 'utils/http'
+
+require 'tmpdir'
+
+require 'puppetserver/ca/logger'
+require 'puppetserver/ca/action/prune'
+require 'puppetserver/ca/utils/http_client'
+
+RSpec.describe Puppetserver::Ca::Action::Prune do
+  include Utils::SSL
+
+  let(:stdout) { StringIO.new }
+  let(:stderr) { StringIO.new }
+  let(:logger) { Puppetserver::Ca::Logger.new(:info, stdout, stderr) }
+
+  subject { Puppetserver::Ca::Action::Prune.new(logger) }
+
+  describe 'flags' do
+    it 'take a custom path to the puppet.conf' do
+      result, maybe_code = subject.parse(['--config', '/dev/tcp/example.com'])
```

Comment:
(nit) I think it would be clearer to call this `maybe_exit_code` (or even just `exit_code`) - at first I was like, what code?? 😅

Reply:
Lol this was a copy and paste from the other test file.

Reply:
ahh, if we use that elsewhere, then I'm fine leaving it as-is

Reply:
but I do agree with what you said about `maybe_code`, I was confused too.